velox/expression/SignatureBinder.cpp

```diff
@@ -24,6 +24,59 @@ namespace {
 bool isAny(const TypeSignature& typeSignature) {
   return typeSignature.baseType() == "any";
 }
+
+bool isNumber(const std::string& str) {
```

Comment:
nit: Would you add a small comment for this function to clarify that it returns true only if 'str' contains digits. Since "number" refers to a broad set of values including 1.23 and -40, but the function supports only positive integers, perhaps, rename the function to isPositiveInteger or isAllDigits.